projects/batfish/src/test/java/org/batfish/main/BatfishTestUtils.java

```diff
@@ -209,4 +257,15 @@ public static Batfish getBatfish(
     }
     return initBatfish(configurations, tempFolder);
   }
+
+  private static SortedMap<String, String> readTestrigResources(
+      String testrigResourcePrefix, String subfolder, String[] filenames) {
```

Comment:
suggest making filenames a `String...` so that you can use the function more manually in the future.